Golden_Path.postman_collection.json

```diff
@@ -1,7 +1,7 @@
 {
 	"info": {
-		"_postman_id": "1e21d368-aee7-4fbd-894a-1222c8ab37d7",
-		"name": "Golden_Path",
+		"_postman_id": "09a47d32-319f-4ecc-8cdc-61a1e606585b",
+		"name": "Golden_Path copy",
```

Comment:
Please remove "copy" from the collection name

Reply:
@ndonnan . done, adjusted in a new cleaned up PR

Reply:
adjusted in a follow-up PR: https://github.com/mojaloop/postman/pull/100